### PR TITLE
Rearrange the structure of the README

### PR DESCRIPTION
**Pull request by alo7lika**
#285 
Changes Made and Rearranged Structure of the README

- Table of Contents Addition
A table of contents was added to allow easier navigation through the README, helping users find specific sections quickly.

- Revised Project Overview
The description of the project was made clearer, explaining its purpose, core features, and significance. This provides a concise introduction to the project.

- Installation and Setup Instructions
The installation and setup instructions were expanded to guide users through the process of running the project locally. This includes the prerequisites, installation steps, and any necessary configuration.

- Enhanced Usage Section
This section now includes clearer instructions on how to use the platform, including navigating the site and interacting with key features such as table booking and exploring events. An API section was introduced, listing all available endpoints along with HTTP methods and descriptions.

- Detailed Contribution Guidelines
The contribution guidelines were improved to encourage open-source participation, explaining how users can create issues, submit pull requests, and contribute to the repository in a structured way.

- Contributors Section Enhancement
The contributors section was reorganized into a table format, showcasing the contributors' GitHub avatars and names beneath each profile link. This creates a visually engaging and organized presentation, accompanied by a note of appreciation to the contributors.

- Contact Section Expansion
The contact section was updated with multiple ways to reach out, including email, social media channels, and GitHub issues. Additional communication platforms such as Discord, Instagram, and Facebook were introduced to provide more ways for community engagement.

- Support Section
A dedicated support section was added, encouraging users to support the project either by becoming a sponsor or by giving the repository a star on GitHub. This section uses clear and concise calls to action to promote project backing.

## Summary by CodeRabbit

- **Documentation**
	- Enhanced clarity and structure of the `README.md` file.
	- Added a "Table of Contents" for easier navigation.
	- Updated project overview and installation instructions for better understanding.
	- Introduced a "Docker Setup" section for Docker users.
	- Expanded sections on "Usage," "API Documentation," "Contributing," and "Support."
	- Added a "Contact Us" section with links for community engagement and feedback.

### CODE REVIEW

**Comment by alo7lika** [comment]
@RamakrushnaBiswal  the task has been completed. Kindly review it.
Thank you!

File: README.md [hunk 1]
```diff
@@ -1,116 +1,136 @@
...
+
+## 🔍 API Documentation
+The PlayCafe website communicates with a backend API to manage data. Here’s a brief overview of available API endpoints:
+
+| HTTP Method | Endpoint       | Description                              |
+|-------------|----------------|------------------------------------------|
+| GET         | `/api/games`   | Fetches the list of available games.     |
+| POST        | `/api/bookings`| Books a table for an event.              |
+| GET         | `/api/events`  | Retrieves upcoming events.               |
+| POST        | `/api/contact` | Sends a message through the contact form.|
```
Comment (RamakrushnaBiswal) [inline README.md:111]: @alo7lika add other api as well

File: README.md [hunk 2]
```diff
@@ -1,116 +1,136 @@
...
 <img src="https://github.com/user-attachments/assets/f7abbbe7-1950-4bf9-8456-c812ca630acf" style="width:400px">
 
 ## 🌟 Mentors
 Special thanks to our amazing mentors who are guiding this project! 🙌
 
-- **[to be annouce]** - Full Stack Developer
-
-## 📄 License
-   This project is licensed under the MIT License. See the [LICENSE](https://github.com/RamakrushnaBiswal/PlayCafe/blob/main/LICENSE) file for more details.
+- **[to be announced]** - Full Stack Developer
```
Comment (RamakrushnaBiswal) [inline README.md:127]: @alo7lika here add @Shiva-Bajpai as a mentor

**Comment by RamakrushnaBiswal** [comment]
@alo7lika also add a contact section and give it
linkedin:- https://www.linkedin.com/in/ramakrushna-biswal/
Discord:-https://discord.gg/Jh3bWQ7FRN
Gmail:-ramakrushnabunty@gmail.com

**Comment by alo7lika** [comment]
@RamakrushnaBiswal , Done all the requested changes you have asked for.

**Comment by alo7lika** [comment]
> @RamakrushnaBiswal , Done all the requested changes you have asked for.

Also, add the labels GSSOC EXT 24 AND HACKTOBERFEST ON THE PR

File: README.md [hunk 3]
```diff
@@ -381,9 +418,35 @@ Special thanks to our amazing mentors who are guiding this project! 🙌
...
 
-## ⭐Support 
+## ⭐ Support
+
+Your support keeps us going! Here's how you can help:
+
+- **💰 Sponsor us**: Become a sponsor and help us keep growing. [Become a Sponsor](https://github.com/sponsors/RamakrushnaBiswal).
+- **⭐ Star our Repo**: Show some love by starring the repository. Your stars motivate us to continue developing and improving!
+
+Thank you for supporting **HelpOps-Hub**! 💖
```
Comment (RamakrushnaBiswal) [inline README.md:429]: HelpOpps hub ??
@alo7lika

File: README.md [hunk 4]
```diff
@@ -1,116 +1,153 @@
...
-   This project is licensed under the MIT License. See the [LICENSE](https://github.com/RamakrushnaBiswal/PlayCafe/blob/main/LICENSE) file for more details.
+Feel free to reach out for guidance and support throughout the development process!
 
-##  👀 Our Contributors
+  
+## 👥 Our Contributors
 
-- We extend our heartfelt gratitude for your invaluable contribution to our project! Your efforts play a pivotal role in elevating HelpOps-Hub to greater heights.
-- Make sure you show some love by giving ⭐ to our repository
+We extend our heartfelt gratitude to all the amazing contributors who have made this project what it is today. Your efforts are driving **HelpOps-Hub** to new heights! 🚀
```
Comment (RamakrushnaBiswal) [inline README.md:148]: Here also @alo7lika HelpOpps hub

Comment (RamakrushnaBiswal) [reply README.md:148]: add PlayCafe

**Comment by alo7lika** [comment]
done @RamakrushnaBiswal.

**Review by RamakrushnaBiswal — APPROVED** [review]

**Pull request merged**